Flight.cpp

```diff
@@ -56,6 +61,7 @@ class AirlineCalculator{
     virtual float getEconomyOpCost (float d) const {
         return 0.0;
     }
+
     virtual float getPremiumOpCost (float d) const {
         return 25.0;
```

Comment:
all const param should be `constexpr double`

Reply:
Why we should use double instead of float? Don't need that more precision.

Reply:
Then constexpr float. The key point is that you can not leave a number there which is not compile friendly. Let's keep high code quality. Forget about all you learn in Samsung.